clean.go

```diff
@@ -156,7 +156,7 @@ func cleanAUR(ctx context.Context, keepInstalled, keepCurrent, removeAll bool, d
 
 		err = os.RemoveAll(filepath.Join(config.BuildDir, file.Name()))
 		if err != nil {
-			return nil
+			return err
```

Comment:
Updated. I've maintained the `return err` inside `cleanAUR` (since logically the operation fails), and added the `text.Warnln`+continue logic at the caller (`syncClean`) instead.

Note that this also makes the other possible errors from `cleanAUR` (e.g.  fetching from AUR failed) non-blocking. Let me know if that sounds good to you.

Reply:
Sounds good to me, done. This way yay will still clean all top-level entries in the cache it can, and notify the user when it can't.